tox.ini

```diff
@@ -60,7 +60,7 @@ commands =
 [testenv:scenario]
 description = Run scenario tests
 deps =
-    pytest
+    pytest<8.2.0 # https://github.com/pytest-dev/pytest/issues/12263
```

Comment:
I think we only need this in integration, it's a matter of incompatibility with pytest-operator. The other envs should be unaffected.